src/include/migraphx/op/quant_convolution.hpp

```diff
@@ -75,7 +75,7 @@ struct quant_convolution
 
     shape normalize_compute_shape(std::vector<shape> inputs) const
     {
-        check_shapes{inputs, *this}.has(2).same_type().same_ndims().min_ndims(3);
+        check_shapes{inputs, *this}.has(2).same_ndims().min_ndims(3);
```

Comment:
If we want to support different types, the compute method needs to be updated to use a different visit for input and weights(instead of using visit_all for them).